.github/workflows/workflow.yml

```diff
@@ -22,7 +22,9 @@ jobs:
       fail-fast: false
     steps:
       - name: Git checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
+        with:
+          depth: 0
```

Comment:
Note: the problem is with our `utils.git`. This utility retrieve git information.
To test it, we are hardcoding a specific old commit of `@netlify/build`: https://github.com/netlify/build/blob/6621f9d44dd211f6ce3713a0319b08b1dcc59d7b/packages/git-utils/tests/main.js#L7.
That's a bad back, which seems to break with `actions/checkout@v2`.

Reply:
Yes, saw this hence trying the `fetch-depth` as a quick fix.
We can also keep the issue open and close this PR if we want to use the proper solution for the tests.

Reply:
Great solution! This is fixing it.
I have added https://github.com/netlify/build/issues/2013 to track improving those tests.